docs/source/ja/_toctree.yml

```diff
@@ -29,9 +29,77 @@
     title: LLM を使用した生成
   title: Tutorials
 - sections:
-  - local: generation_strategies
-    title: 生成戦略をカスタマイズする
-  title: Generation
+  # - isExpanded: false
```

Comment:
Let's not add all these until they're actually ready. This makes it easier to review and fix any issues in the `toctree`.